todo_app/get_a_board.py

```diff
@@ -0,0 +1,15 @@
+import requests
```

Comment:
I suspect this was used for testing - that's a sensible approach, but we might as well delete it now since it appears to be no longer used. We can always recover it with Git later